tests/ui/suggestions/chain-method-call-mutation-in-place.stderr

```diff
@@ -18,7 +18,10 @@ error[E0599]: no method named `sort` found for unit type `()` in the current sco
   --> $DIR/chain-method-call-mutation-in-place.rs:3:72
    |
 LL |     vec![1, 2, 3].into_iter().collect::<Vec<i32>>().sort_by_key(|i| i).sort();
-   |                                                                        ^^^^ method not found in `()`
+   |                   -----------                       ------------------ ^^^^ method not found in `()`
+   |                   |                                 |
+   |                   |                                 method `sort` is available on `&mut [i32]`
+   |                   method `sort` is available on `Vec<i32>`
```

Comment:
@estebank how do I get rid of this pesky one? Sort is available on `Vec` but since `sort_by_key`s return is unit, the check up method chain seems irrelevant.

Reply:
The spans here seem to be shifted by one. The first span should be pointing at the `vec![1, 2, 3]` and the second one to the `.collect::<Vec<i32>>()`. The logic to identify the call is correct, you just likely need to point at the receiver of the call, not the segment.